bugsnag-plugin-android-anr/src/main/jni/bsg_unwind.h

```diff
@@ -1 +0,0 @@
-../../../../bugsnag-plugin-android-ndk/src/main/jni/bsg_unwind.h
```

Comment:
is this intentional? I thought this was required to access restricted functions in the ANR module.

Reply:
It was the only way to keep the definition in one file, because it's used by the internal API for stack unwind.

I had to link it, otherwise we'd have to make two copies of it and remember to keep them in sync.